list

```diff
@@ -1157,6 +1168,7 @@ so.arte
 soc.cr
 soch.us
 social.ora.cl
+socx.in
```

Comment:
@MrMoncler do you also have an example for socx.in? Though it looks like it's already in some blacklists. References:

[`~/repos/chkdomain $ ./chkdm socx.in`](https://github.com/PeterDaveHello/chkdomain)

> You are checking domain: socx.in
> 
> Running dig/nslookup over 11 nofilter DNS:
>  - AdGuard (94.140.14.140) ... OK! (20.43.132.131)
>  - Cloudflare (1.1.1.1) ... OK! (20.43.132.131)
>  - dns0.eu (193.110.81.254) ... OK! (20.43.132.131)
>  - Freenom World (80.80.81.81) ... Failed!
>    ** server can't find socx.in: SERVFAIL
>  - Gcore (95.85.95.85) ... OK! (20.43.132.131)
>  - Google (8.8.8.8) ... OK! (20.43.132.131)
>  - Hinet (168.95.1.1) ... OK! (20.43.132.131)
>  - OpenDNS (208.67.222.2) ... OK! (20.43.132.131)
>  - Quad9 (9.9.9.10) ... OK! (20.43.132.131)
>  - UltraDNS (64.6.64.6) ... OK! (20.43.132.131)
>  - Yandex (77.88.8.1) ... OK! (20.43.132.131)
> 
> Running dig/nslookup over 11 secure DNS:
>  - CleanBrowsing (185.228.168.9) ... Failed!
>    ** server can't find socx.in: NXDOMAIN
>  - Cloudflare (1.1.1.2) ... OK! (20.43.132.131)
>  - Comodo (8.26.56.26) ... OK! (20.43.132.131)
>  - CONTROL D (76.76.2.1) ... Failed!
>    Address: 0.0.0.0
>    Address: ::
>  - dns0.eu (193.110.81.0) ... Failed!
>    ** server can't find socx.in: NXDOMAIN
>  - OpenDNS (208.67.222.222) ... OK! (20.43.132.131)
>  - Quad101 (101.101.101.101) ... OK! (20.43.132.131)
>  - Quad9 (9.9.9.9) ... OK! (20.43.132.131)
>  - SafeDNS (195.46.39.39) ... OK! (20.43.132.131)
>  - UltraDNS (156.154.70.2) ... OK! (20.43.132.131)
>  - Yandex (77.88.8.2) ... OK! (20.43.132.131)
> 
> Running dig/nslookup over 6 AD(and tracker)-blocking DNS:
>  - AdGuard (94.140.14.14) ... OK! (20.43.132.131)
>  - AhaDNS (5.2.75.75) ... Failed!
>    ** server can't find socx.in: REFUSED
>  - CONTROL D (76.76.2.2) ... Failed!
>    Address: 0.0.0.0
>    Address: ::
>  - dnsforge.de (176.9.93.198) ... OK! (20.43.132.131)
>  - OVPN (192.165.9.157) ... OK! (20.43.132.131)
>  - Tiarap (188.166.206.224) ... OK! (20.43.132.131)
> 
> Running nslookup default DNS (127.0.0.1):
>  - 127.0.0.1 ... Failed!
>    Address: 0.0.0.0
> 
> Get more intels about this domain from:
> AlienVault Open Threat Exchange
>   - https://otx.alienvault.com/indicator/domain/socx.in
> Bitdefender TrafficLight
>   - https://trafficlight.bitdefender.com/info/?url=https%3A%2F%2Fsocx.in
> Google Safe Browsing
>   - https://transparencyreport.google.com/safe-browsing/search?url=socx.in
> Kaspersky Threat Intelligence Portal
>   - https://opentip.kaspersky.com/socx.in?tab=web
> McAfee SiteAdvisor
>   - https://siteadvisor.com/sitereport.html?url=socx.in
> Norton Safe Web
>   - https://safeweb.norton.com/report/show?url=socx.in
> OpenDNS
>   - https://domain.opendns.com/socx.in
> URLVoid
>   - https://www.urlvoid.com/scan/socx.in/
> urlscan.io
>   - https://urlscan.io/domain/socx.in
> VirusTotal
>   - https://www.virustotal.com/en/domain/socx.in/information/
> Yandex Site safety report
>   - https://yandex.com/safety/?l10n=en&url=socx.in

Reply:
https://socx.in/69zl81 - Target: `https://github.com/PeterDaveHello/url-shorteners`

Reply:
Thanks.